### ISSUE TITLE
Promotion de Publicodes 

### ISSUE THREAD

**Issue by penelopel** [betagouv/mon-entreprise#1220]
**Objectif : potentiellement en faire un projet indépendant porté par la DINUM.**

Pour que le projet intéresse la DINUM, il faut : 

1. Montrer que l'outil est utilisé par plusieurs start-ups d'Etat, qui travaillent dans des ministères différents,
2. Monter un dossier pour montrer le potentiel d'exploitation supérieure dans les SET (et hors SET) et le faire porter par un sponsor (ex: co-animateur de la DINUM) 
NB : bien montrer la différence avec les cas d'Open Fisca et de Démarches Simplifiées. 

Micro-tâches : 

- [ ] Mise à jour du site, remplacer futur.eco par https://ecolab.ademe.fr/impactcarbone sur https://publi.codes/

# Faire utiliser Publicodes à un maximum de SET 

## Etat des lieux
Pour le moment, Publicodes est utilisé par : 

- Mon-entreprise.fr (17 simulateurs) - ACOSS / Ministère de l'Action et des Comptes publics et Ministère des Solidarités et de la Santé.
- Ecolab (2 simulateurs) - ADEME / Ministère de la Transition Ecologique
[A compléter]

En discussion : 
- Egalim - Direction générale de l'alimentation / Ministère de l'Agriculture et de l'Alimentation 

## Comment accroître cette exposition ? 

- [ ] Faire un résumé sympa de ce qu'il est possible de faire avec Publi.codes (à positionner comme "composante unique" ou comme "complément" de produit) qui ne parle pas qu'aux dev (pour que les autres membres des équipes puissent identifier aussi un intérêt potentiel)
- [ ] Lancer le sujet sur le Slack en proposant un échange avec ceux qui peuvent être intéressés
- [ ] Trouver le bon endroit du Gitbook des SE où on pourrait parler de Publi.codes (ex: [choix des technologies](https://doc.incubateur.net/startups/developpement/choisir-un-langage-de-programmation) et l'y proposer  ([comment faire](https://doc.incubateur.net/startups/contribuer/comment-faire)) - l'idée est de créer un "réflexe" Publi.codes pour le futur et que tout le monde y soit exposé d'une manière ou d'une autre à un moment 
- [ ] Regarder [les SE en construction ](https://beta.gouv.fr/startups/#construction) et contacter sur Slack si affinités possibles avec Publicodes 
- [ ] Regarder [les SE en investigation ](https://beta.gouv.fr/startups/#investigation) et contacter si affinités possibles avec Publicodes 

## Comment s'assurer de la bonne intégration 
- être en maxi-support des équipes qui veulent l'utiliser
- documenter leurs problèmes/questions qu'ils décident ou non d'utiliser publicodes

# Suite 
Une fois qu'on aura fait tout ça, on pourra écrire la suite :)

**Comment by mquandalle** [comment]
On a aussi prévu une session en décembre avec @laem pour traduire en anglais le site et les mots clés du langage en lui-même, ce qui devrait élargir le nombre d'utilisateurs potentiels.

**Comment by penelopel** [comment]
Yes! (tu as vu je m'adapte hehe) On en a parlé au stand-up ce matin d'où l'issue pour nourrir votre journée spéciale Publicodes ;)

**Comment by lajarre** [comment]
Pour complémenter sur la partie retours: j'aurais carrément dit qu'on devrait faire du user testing. En gros recruter des devs d'autres startups d'Etat pour une petite session où on on observe l'utilisation de publicodes, etc.

Je sais pas si c'est qqch qui a déjà été fait?

On pourrait nourrir ce processus de tests utilisateurs avec les contacts qu'on prend pour essayer de promouvoir Publicodes auprès des SET, comme décrit ci-dessus.

**Comment by johangirod** [comment]
> Je sais pas si c'est qqch qui a déjà été fait?

Si, on a déjà eu l'occasion de faire une session de pair, mais c'était il y a trèèèès longtemps. Ca serait pas mal de le refaire en effet.

J'en profite pour mettre un lien vers #1156 qui propose une définition de publicodes qui peut être porteuse.

**Comment by laem** [comment]
Sur Ecolab je travaille avec 2 ou 3 profils non-dev qui modifient du YAML. C'est sans surprise pas évident, mais cela dit ils arrivent quand même à rédiger du pseudo-code qui aide un développeur à reprendre et faire compiler le code. Donc @lajarre des tests utilisateurs sur un public non-dev, c'est déjà parti :) 

Ecolab est potentiellement en voie d'être décliné par une demi-douzaine d'acteurs français (entreprises, assos, collectivités) et des acteurs internationaux (la Belgique, seul possibilité tant que le système n'est que francophone).

**Comment by lajarre** [comment]
Cool @laem je ne me rendais pas tout-à-fait compte de l'étendue du projet!

Ceci dit, je ne suis pas sûr d'avoir compris quels sont les personas qu'on vise: devs qui ont besoin d'un langage avec une syntaxe pratique pour faire des règles, pms qui ont besoin d'un langage abordable, pms qui ont besoin de se baser sur des règles existantes, devs open-source qui veulent contribuer à une base de règles… C'est quoi selon vous?

D'après moi, il faudrait faire une sorte d'étude de marché (même si c'est limité à beta) pour savoir de qui on parle et de quels besoins. Ceci dit vous l'avez peut-être déjà faite du moins dans votre tête?

**Comment by penelopel** [comment]
Clément (Renault) avait testé : ses retours critiques, c'était que la doc n'était pas hyper claire et que le modèle d'intégration était compliqué (sinon il voyait un potentiel énorme hein c'était son premier retour). On pourrait organiser une session avec lui de test-utilisateur, il était chaud. 
Niveau profil je pense qu'il n'est pas dev mais quand même très tech et qu'il pourrait aussi nous aider sur les besoins/études de marché, d'autant qu'il est sur pas mal de SET.

**Comment by penelopel** [comment]
Les premiers tests d'Alex: https://pad.incubateur.net/FZ1ivRtNQlqMmDj3-y-9xw#
